### ISSUE TITLE
macOS hotkey for Go to Home View is surprising, should use Command instead of Control

### ISSUE THREAD

**Issue by lilyball** [vector-im/element-web#19850]
### Steps to reproduce

1. Run the app on macOS
2. Press ⇧⌘H

#### What did you expect?
It should have gone to the Home View

#### What happened instead?
Nothing at all, because the shortcut is actually ⌃⇧H

The other shortcuts use the Command key as appropriate, this is the only one that uses Control inappropriately. The other hotkeys using Control are modifiers on the arrow keys, which is fine.

### Operating system

macOS

### Application version

Version 1.9.5

### Will you send logs?

No

**Comment by kittykat** [comment]
Yes, this should be resolved by updating the shortcut to ⇧⌘H - issue is ready for development :+1:

**Comment by aj-ya** [comment]
Hey @kittykat ! I would like to work on this issue.

**Comment by aj-ya** [comment]
Hey! @SimonBrandner Thanks for assigning me to this issue! After going through those files and the issue, I believe I have to bind the 'Go to Home' view to the following key combinations.
on Mac : SHIFT + ⌘ + H ( presently CTRL+SHIFT+H )
on PC:  CTRL + ALT + H  ( presently CTRL+ALT+H )
Please give me a thumbs up, if I'm right!

**Comment by SimonBrandner** [comment]
> Hey! @SimonBrandner Thanks for assigning me to this issue! After going through those files and the issue, I believe I have to bind the 'Go to Home' view to the following key combinations. on Mac : SHIFT + ⌘ + H ( presently CTRL+SHIFT+H ) on PC: CTRL + ALT + H ( presently CTRL+ALT+H ) Please give me a thumbs up, if I'm right!

Sounds good :+1:

**Comment by aj-ya** [comment]
Hey! I've opened a PR regarding this issue. please check it out and give suggestions if needed.
